docs/Основные блоки и устройство компьютера/Устройства ввода информации.md

```diff
@@ -0,0 +1,37 @@
+# Устройство ввода информации
+
+**Устройства ввода** — технические устройства, которые преобразуют сигнал в данные, пригодные для обработки на компьютере или управления им.
+
+## Клавиатура
+
+**Клавиатура** — компьютерное устройство, которое располагается перед экраном дисплея и служит для набора текстов и управления компьютером с помощью клавиш, находящихся на клавиатуре.
+
+*Клавиатура* позволяет вводить в компьютер числовую и текстовую информацию, а также различные команды и данные.
+
+## Компьютерная мышь
+
+**Компьютерная мышь** — это устройство ввода информации, применяемое в компьютерных системах для управления курсором на экране. Она позволяет пользователю перемещать и выбирать объекты на экране, а также выполнить другие действия, связанные с взаимодействием с графическим интерфейсом пользователя.
+
+## Веб-камера
+
+Еще одним устройством ввода информации является веб-камера. **Веб-камера** – это малоразмерная цифровая видео или фотокамера, способная в реальном времени фиксировать изображения, предназначенные для дальнейшей передачи по сети Интернет. 
+
+*Веб-камеры*, доставляющие изображения через Интернет, закачивают изображения на веб-сервер либо по запросу, либо непрерывно, либо через регулярные промежутки времени. Это достигается путем подключения камеры к компьютеру или благодаря возможностям самой камеры.
+
+## Микрофон
+
+**Микрофон** – это устройство, которое используется для ввода звуковой информации. Он преобразует звуковые волны в электрические сигналы, которые затем могут быть записаны или переданы на компьютер или другое устройство. Микрофоны могут быть подключены к компьютеру через аналоговые аудио-разъемы или USB-порты.
+
+## Сканер
+
+**Сканер** – это устройство, которое позволяет считывать изображение с листа бумаги, при этом оптический сигнал, получаемый при сканировании лазерным лучом изображения, преобразуется в электрический. Он передается по кабелю в компьютер и преобразуется в двоичный код.
+
+*Сканеры бывают:*
+-планшетные (позволяют разместить на специальной панели лист бумаги);
+-ручные (текстовые). Представляют собой ручку, при помощи которой можно сканировать лист бумаги построчно. Такие сканеры также используются при считывании штрих-кодов.
+
+## Источники
```

Comment:
[ch-md-error-separate-blocks](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-separate-blocks)